app/resources/device.py

```diff
@@ -130,17 +130,15 @@ def change_name(data: dict, user: str) -> dict:
     return device.serialize
 
 
-@m.user_endpoint(path=["device", "delete"], requires={
-    "device_uuid": UUID()
-})
+@m.user_endpoint(path=["device", "delete"], requires={"device_uuid": UUID()})
 def delete(data: dict, user: str) -> dict:
```

Comment:
stop and delete all services associated with this device

Reply:
done